samples/ironside_se/protectedmem_periphconf/README.rst

```diff
@@ -0,0 +1,173 @@
+.. _protectedmem_periphconf_sample:
+
+Protected Memory with PERIPHCONF Partition
+##########################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates how to protect the ``periphconf_partition`` using UICR.PROTECTEDMEM. The sample shows that when protected memory is modified, the integrity check fails on the next boot, causing IronSide SE to boot a secondary firmware instead of the main application.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+This sample demonstrates:
+
+* How to configure UICR.PROTECTEDMEM to protect a memory region
+* How to relocate the ``periphconf_partition`` using a device tree overlay to be placed right after ``cpuapp_boot_partition``
+* How to configure the PROTECTEDMEM size in Kconfig to cover both ``cpuapp_boot_partition`` and ``periphconf_partition``
+* How to configure a secondary firmware that boots when PROTECTEDMEM integrity check fails
```

Comment:
The user does not configure that secondary firmware is booted when PROTECTEDMEM integrity check fails. With this phrasing it sounds like this is a configuration.
```suggestion
* How to configure a secondary firmware.
```